platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -148,6 +148,12 @@ typedef struct ODP_ALIGNED_CACHE odp_packet_hdr_t {
 	/* LSO profile index */
 	uint8_t lso_profile_idx;
 
+	/* Tx completion event queue */
+	odp_queue_t tx_compl_queue;
+
+	/* Tx completion event mode */
+	odp_packet_tx_compl_mode_t tx_compl_mode;
```

Comment:
Packet header struct should be kept as compact (small and packed) as possible. tx_compl_mode enum should not be needed, tx_compl flag in output flags should be enough. dst_queue could be reused for TX compl, since it's currently used only on packet input side (by classifier).

Reply:
`dst_queue` it is, enum ditched as well in v2.